suite-common/suite-config/src/settings.ts

```diff
@@ -1,5 +1,5 @@
 export const settingsCommonConfig = {
     MAX_ACCOUNTS: 10,
     FRESH_ADDRESS_LIMIT: 20,
-    TXS_PER_PAGE: 25,
+    TXS_PER_PAGE: 2,
```

Comment:
TXs per page setting mentioned in my comment.

Reply:
I haven't found any other source of this change than this branch, but it definitely shouldn't be there as it affects all the coins.

Reply:
fix pushed 👍

Reply:
Probably got missed when dividing the changes for suite :thinking: We have set it to 2 temporarily when we had just free plan for solana quicknode. A fix was merged a while go which reverted it https://github.com/trezor/trezor-suite/pull/9915. @memoriesadrift pls make sure that the way you reverted it matches the original revert.